CONTRIBUTING.md

```diff
@@ -3,6 +3,16 @@
 Welcome to The Lounge, it's great to have you here! We thank you in advance for
 your contributions.
 
+### I have found a security vulnerability
+
+- If you think you discovered a security vulnerability, **please do not open a
+  public issue about it on GitHub.** Contact us privately first, in a
+  [responsible disclosure](https://en.wikipedia.org/wiki/Responsible_disclosure)
+  manner.
+- On IRC, contact in private message anyone who is a voiced user on our Freenode
+  channel `#thelounge`.
+<!-- - Or send us an email at <mailto:security@thelounge.chat>.-->
```

Comment:
this is not set up so I just added it as a template, but it would be nice if this emailed us @thelounge/maintainers. Opt-in of course, if some maintainers do not want to get emails. I'm happy to be on the list, personally.

Reply:
Happy to be on the list.